gigadb/app/configurator/tests/makeDotSecrets.bats

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bats
+
+source ./src/dotfiles.sh
+
+teardown () {
+	if [ -f tests/.secrets ];then
+		rm tests/.secrets
+	fi
+}
+
+@test "source .secrets if already exists" {
+    echo "foo=bar" > tests/.secrets
+    run makeDotSecrets tests
+    [ "$output" = "Sourcing secrets" ]
+}
+
+# bats test_tags=TODO
+@test "can generate .secret" {
+    skip
+}
```

Comment:
This `TODO` test can be fixed with this patch:
```
@test "can generate .secrets" {
    run makeDotSecrets tests
    [ -s tests/.secrets ] # check if .secrets exists and its size is not zero
}
```

Reply:
Hi @kencho51,

Good call. I have added the changes.